src/Components/Footer.js

```diff
@@ -0,0 +1,13 @@
+// import React from 'react';
+// import './Footer.css';
+
+// const Footer = () => (
+//   <>
+//     <div className="footer">
+//         <p className='foot-p'>&copy; 2022, E-commerce Website for your online sales.</p>
+//           <span>Tel: +393298592707</span>
+//           <span>isigba.ambrose94@gmail.com</span>
+//     </div>
+//   </>
+// );
+// export default Footer;
```

Comment:
- [ ] You have commented on all of your code for the footer. As the previous reviewer suggested, You can work on it. :+1: